6-magic.md

```diff
@@ -491,38 +309,18 @@ in the close combat phase she will score 1 automatic hit and then the opponent w
-Necromancer may raise him to fight as a Zombie in his servitude.
-The dead Hero retains his characteristics and all his weapons and armor but may not use any other equipment or skills.
-He may no longer run, counts as a Henchman group on his own, and may not gain additional experience. This spell
-always succeeds (rules for Henchmen and experience are described later). The new Zombie follows all the normal
-Zombie rules (immune to poison, causes fear) except for retaining his profile, weapons and armor. |
+| 1 | Lifestealer | 9 | Direct Damage. The Necromancer sucks out the very essence of life from their victim, stealing its vigour for himself. You may choose a single model within 6". The target suffers a wound with no armour save allowed and the Necromancer gains an extra wound for the duration of the battle. This may take the Necromancer’s Wounds above his original maximum value. |
+| 2 | Re-Animation | 5 | Augment. At the spoken command of the Necromancer, the dead rise to fight again. One Zombie that went out of action during the last hand-to-hand combat or Shooting phase immediately returns to the battle. Place the model within 6" of the Necromancer. The model cannot be placed straight into hand-to-hand combat with an enemy model. |
+| 3 | Death vision | 6 | Augment. The Necromancer calls on the power of Necromancy to reveal the moment of his enemies’ death. The Necromancer causes Terror for the duration of the battle. |
+| 4 | Romejo's Awakening | 8 | Direct Damage. The Necromancer whispers to the sleeping dead to rise up from the ground and seize their enemies. Choose one enemy model within 12". The model must immediately roll equal to or less than their Strength on a D6 or the dead emerging from the earth will rend him with supernatural power. If they fail, you may roll on the Injury table to see what happens to the unfortunate warrior. Has no effect on creatures Large or greater. |
+| 5 | Call of Vanhel | 10 | Augment. The Necromancer summons the world of the dead to invigorate their Undead servants.
+All Undead allies within 6" of the Necromancer immediately make a free move - including charging. |
+6 Spell of Awakening | Auto | The Necromancer calls the soul of a slain Hero back to his body and enslaves him with corrupt magic. If an enemy Hero is killed (i.e., your opponent rolls 11-16 on the Serious Injury chart after the battle) then the Necromancer may raise him to fight as a Zombie in his servitude. The dead Hero retains his characteristics and all his weapons and armor but may not use any other equipment or skills. He may no longer run, counts as a Henchman group on his own, and may not gain additional experience. This spell always succeeds (rules for Henchmen and experience are described later). The new Zombie follows all the normal Zombie rules (immune to poison, causes fear) except for retaining his profile, weapons and armor. |
```

Comment:
```suggestion
| 6 | Spell of Awakening | Auto | The Necromancer calls the soul of a slain Hero back to his body and enslaves him with corrupt magic. If an enemy Hero is killed (i.e., your opponent rolls 11-16 on the Serious Injury chart after the battle) then the Necromancer may raise him to fight as a Zombie in his servitude. The dead Hero retains his characteristics and all his weapons and armor but may not use any other equipment or skills. He may no longer run, counts as a Henchman group on his own, and may not gain additional experience. This spell always succeeds (rules for Henchmen and experience are described later). The new Zombie follows all the normal Zombie rules (immune to poison, causes fear) except for retaining his profile, weapons and armor. |
```

Reply:
Thanks for the correction - also I realise I forgot to change the He/They problem for that spell.

Reply:
I've updated, good spot